### PR TITLE
Remove Authenticators

### PR DESCRIPTION
**Pull request by BurntVal**
First iteration for removing authenticators.

https://github.com/burnt-labs/xion.js/assets/117188229/fed8edbf-9e8e-4fc8-b13b-eda15041a436

EDIT (Added trash icon per design spec):

Screenshot 2024-05-15 at 5 55 39 PM

### CODE REVIEW

File: apps/abstraxion-dashboard/components/ModalViews/RemoveAuthenticator/RemoveAuthenticatorForm.tsx [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+            {handleAuthenticatorLabels(
+              authenticator.type.toUpperCase() as authenticatorTypes,
+            )}
+          </p>
+        </div>
+      </div>
+    );
+  };
+
+  function postAddFunction() {
```
Comment (BurntNerve) [inline apps/abstraxion-dashboard/components/ModalViews/RemoveAuthenticator/RemoveAuthenticatorForm.tsx:127]: What does this mean?

File: apps/abstraxion-dashboard/components/ModalViews/RemoveAuthenticator/RemoveAuthenticatorForm.tsx [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+      if (!client) {
+        throw new Error("No client found.");
+      }
+
+      if (!abstractAccount) {
+        throw new Error("No account found.");
+      }
+
+      if (abstractAccount.authenticators.nodes.length <= 1) {
+        throw new Error("We cannot allow this operation.");
```
Comment (BurntNerve) [inline apps/abstraxion-dashboard/components/ModalViews/RemoveAuthenticator/RemoveAuthenticatorForm.tsx:149]: If we're going to throw this error can we flesh out out the message to be more clear why this action is forbidden?

**Review by BurntNerve — APPROVED** [review]
Small comments, looks good!

**Review by BurntNerve — APPROVED** [review]
looks good!

**Review by justinbarry — APPROVED** [review]

**Pull request merged**